docs/basic-features/data-fetching/get-static-props.md

```diff
@@ -84,7 +84,7 @@ When a page with `getStaticProps` is pre-rendered at build time, in addition to
 
 This JSON file will be used in client-side routing through [`next/link`](/docs/api-reference/next/link.md) or [`next/router`](/docs/api-reference/next/router.md). When you navigate to a page that’s pre-rendered using `getStaticProps`, Next.js fetches this JSON file (pre-computed at build time) and uses it as the props for the page component. This means that client-side page transitions will **not** call `getStaticProps` as only the exported JSON is used.
 
-When using Incremental Static Generation `getStaticProps` will be executed out of band to generate the JSON needed for client-side navigation. You may see this in the form of multiple requests being made for the same page, however, this is intended and has no impact on end-user performance
+When using Incremental Static Generation `getStaticProps` will be executed out of band to generate the JSON needed for client-side navigation. You may see this in the form of multiple requests being made for the same page, however, this is intended and has no impact on end-user performance.
```

Comment:
```suggestion
When using Incremental Static Generation, `getStaticProps` will be executed "out of band" to generate the JSON needed for client-side navigation. You may see this in the form of multiple requests being made for the same page, however, this is intended and has no impact on end-user performance.
```

Reply:
I feel like we need to define what out of band means here - it's not entirely clear.

Your changes look great though 😄

Reply:
Thanks Lee! 

I am not sure what does that means in this context either. Do you have more context that you can share with me?

Reply:
@leerob can you please take a look if the above rephrasing is accurate or not?

Reply:
I like it!

Reply:
Awesome. Thanks!